docs/staking-and-delegation/managing-stake-tao-app.md

```diff
@@ -28,26 +48,400 @@ When the sum is closer to around 1 for the time period specified, it suggest tha
 When the sum is greater than 1 for the time period specified, it suggests that at least some subnets are experiencing buying pressure, which is driving up prices.
 When there is a high sum (e.g. 10+) in a relatively short time frame, it could indicate widespread speculation, possibly even signaling a bubble, or at least high usage/subnet demand.
 
-## Metrics/dTAO Market Overview
+### TAO injected vs TAO Reserves
 
-The dTAO Market Overview panel gives ecosystem users and investors an overview into the many measurements for assessing various important aspects of the state of the overall TAO ecosystem. 
+#### How it works
 
-### Sum of Alpha Prices
+In the TAO ecosystem, each subnet has a liquidity pool composed of TAO. 
```

Comment:
In the TAO ecosystem, each subnet has a liquidity pool composed of TAO and that subnet's token (commonly called as alpha).